lib/src/mock_query.dart

```diff
@@ -93,8 +93,18 @@ class MockQuery extends Mock implements Query {
     return MockQuery(this, (documents) {
       final sortedList = List.of(documents);
       sortedList.sort((d1, d2) {
-        final value1 = d1.data[field] as Comparable;
-        final value2 = d2.data[field];
+        dynamic value1;
```

Comment:
Is it possible to declare this as a Comparable instead of dynamic? If field is a String, we know it's a Comparable. If it's a document ID, it's a String, so it's also a Comparable.